server/routes/title.js

```diff
@@ -1,59 +1,50 @@
 const router = require('express').Router();
+const Promise = require('bluebird');
 
 let Title = require('../../db/title.model');
 const mongoose = require('mongoose');
+const saveEnrolled = require('./enrolled').saveEnrolled;
+const exampleDataGenerator = require('../example.data').exampleDataGenerator;
 
-const dataGeneratorFunction = require('../example.data');
-
-//helper function
-let saveTile = (randomData, cb) => {
-  var titleArray = randomData[0].titleName;
-  var idArray = randomData[0].id;
-  titleArray.forEach((data, index) => {
-    const title = data;
-    const id = String(idArray[index]);
-
-    Title.findOne({title: title}, (err, data) => {
-      if (err) { console.log(err); }
-      if (data) {
-        cb(`${title} already exists in db`);
-      } else {
-        const repo = new Title({
-          id,
-          title
-        });
-        repo.save()
-          .then((title) => cb(title))
-          .catch(err => cb(err));
-      }
-    });
-  });
+let saveTittle = (cb) => {
+  let tittleData = exampleDataGenerator();
+
+  Title.insertMany(tittleData)
+    .then((seededData) => cb(null, seededData))
+    .catch((err => cb(err, null)));
+};
+
+saveTittle = Promise.promisify(saveTittle);
+
+const seed = async (cb) => {
+  saveTittle()
+    .then((seededTittles) => console.log('seededTittles = ', seededTittles))
+    .catch((err) => console.log('ERROR SEEDING TITTLES = ', err));
+
+  await saveEnrolled()
```

Comment:
Since this is an async function you can change it to 'let seededTittles = await saveTittle()', that would eliminate the callback structure.  This might be a good opportunity to try and eliminate some of the .then / promises and replace with await / try & catch blocks.

Reply:
Thank you for the suggestion Tim